cvat-ui/src/components/create-cloud-storage-page/cloud-storage-form.tsx

```diff
@@ -414,6 +415,25 @@ export default function CreateCloudStorageForm(props: Props): JSX.Element {
             );
         }
 
+        if (providerType === ProviderType.AZURE_CONTAINER && credentialsType === CredentialsType.CONNECTION_STRING) {
+            return (
+                <>
+                    <Form.Item
+                        label='Connection string'
+                        name='connection_string'
+                        rules={[{ required: true, message: 'Please, specify your connection string' }]}
+                        {...internalCommonProps}
+                    >
+                        <Input.Password
+                            maxLength={437}
```

Comment:
Why is max length 437 here but in server serializer 440? What is the real maximum length for connection string?